src/components/SignupForm.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+
+const SignupForm: React.FC = () => {
+    const [email, setEmail] = useState('');
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        //TODO: add signup form logic here(Still setting up accounts)
+        console.log('Email submitted:', email);
+    };
+
+    return (
+        <div className="flex flex-col items-center justify-center h-2/4 bg-transparent space-y-8">
+            <h2 className="text-3xl font-bold text-navy-blue drop-shadow-xl">Stay Posted!</h2>
+            <form onSubmit={handleSubmit} className="w-64">
```

Comment:
also maybe flex box so the email and subscribe button are inline with each other

Reply:
doesn't the `flex-col items-center justify-center` do just this already?